cms/envs/common.py

```diff
@@ -228,14 +228,9 @@
 for namespace, template_dirs in lms.envs.common.MAKO_TEMPLATES.iteritems():
     MAKO_TEMPLATES['lms.' + namespace] = template_dirs
 
+# Django templating
 TEMPLATE_DIRS = MAKO_TEMPLATES['main']
```

Comment:
Hmm, settings.TEMPLATE_DIRS is manipulated in lms/startup.py.  It'd be better to unify all this under TEMPLATES I think.